packages/parser/test/fragments/README.md

```diff
@@ -0,0 +1,3 @@
+Fragments of DoenetML code to test the parser.
+
+Fragments `xml-*` come from the `lezer-xml` test cases: https://github.com/lezer-parser/xml/tree/main MIT license
```

Comment:
This comment seems incorrect since xml-doenet-* files don't appear to be standard xml.

Reply:
Good call. Only the `xml-##` ones are from `lezer-xml`